src/gpl/test/BUILD

```diff
@@ -59,3 +59,44 @@ filegroup(
     name = test_name,
     data = [":test_resources"],
 ) for test_name in TESTS]
+
+OPENROAD_COPTS = [
+    "-fexceptions",
+    "-Wno-error",
+    "-Wall",
+    "-Wextra",
+    "-pedantic",
+    "-Wno-cast-qual",  # typically from TCL swigging
+    "-Wno-missing-braces",  # typically from TCL swigging
+    "-Wredundant-decls",
+    "-Wformat-security",
+    "-Wno-sign-compare",
+    "-Wno-unused-parameter",
+    "-Wno-tautological-constant-out-of-range-compare",
+    "-Wno-ctad-maybe-unsupported",
+    "-Wno-nullability-extension",
+    "-fopenmp",
+]
+
+cc_test(
+    name = "mbff_test",
```

Comment:
It is problematic to have bazel only tests.

Reply:
Adding the test to cmake is in progress.  Two prerequisite PRs will come first:

 https://github.com/The-OpenROAD-Project/OpenROAD/pull/8250
 https://github.com/The-OpenROAD-Project/OpenROAD/pull/8252